### PR TITLE
refactor(palette): use v2 primary, secondary colours

### LINKED ISSUE #559: Implement new UI colours

### PR DESCRIPTION
**Pull request by LoneRifle**
## Problem

As part of efforts to migrate to v2 of the design system, update secondary colour palette to v2,
and tweak colour assignments for all references to secondary colours as well as some to primary

- update secondary colours to v2
- rework selector component layout, search bar to line up to v2
- update secondary/primary colour assignments to v2

## Testing

Use staging environment to inspect changes

### CODE REVIEW

**Review by wanlingt — COMMENTED** [review]
image
search placeholder text can't be seen fully, should be `font-size: 16px;` in the
[text styles file](https://github.com/opengovsg/askgovsg/blob/b5fc2ef2fadeda15e6204476c75ca3fdfa8b4b3d/client/src/styles/_text.scss#L193)

**Review by LinHuiqing — COMMENTED** [review]
Screenshot 2021-10-29 at 12 44 40 PM

Colour [here](https://staging.ask.gov.sg/agency/ago?tags=Audit) should be consistently the light grey (secondary 700). Referring to screen on Design Master v2: https://www.figma.com/file/O6lxjjfn04Ow6nn2TMStpk/AskGov-Design-Master-v2?node-id=5864%3A55933

File: client/src/components/SearchBox/SearchBox.component.jsx [hunk 1]
```diff
@@ -136,18 +136,23 @@ const SearchBox = ({
...
+                    mb="8px"
+                    mt="8px"
+                    mr="8px"
+                    borderRadius="4px"
+                    cursor="pointer"
+                    onClick={() => handleSubmit(inputValue)}
                   />
                 ) : null}
                 <Input
                   variant={showSearchIcon ? 'unstyled' : undefined}
```
Comment (wanlingt) [inline client/src/components/SearchBox/SearchBox.component.jsx]:
```suggestion
                  variant="unstyled"
```

to get rid of the white box outlined under Question Title input box

image

**Review by wanlingt — APPROVED** [review]
other than the Question Title comment, lgtm!

**Pull request merged**